code/game/objects/items/weapons/explosives.dm

```diff
@@ -33,11 +33,14 @@
 			else if(role == "changeling")
 				message_say = "FOR THE HIVE!"
 			else if(role == "cultist")
-				message_say = "FOR NARSIE!"
+				message_say = "FOR NAR-SIE!"
+			else if(role == "revolutionary" || role == "head revolutionary")
+				message_say = "VIVA LA REVOLUTION!"
 	user.say(message_say)
 	target = user
 	message_admins("[key_name(user, user.client)](<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A>) suicided with [src.name] at ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)",0,1)
 	explode(get_turf(user))
+	user.gib()
 	return .

```

Comment:
remove this return and the . on this line, it's entirely unnecessary and is technically (no matter how small) adding overhead.